README.md

```diff
@@ -915,6 +915,7 @@ This repository also provides one such platforms where contributers come over an
 | [Drum_kit] (https://github.com/kunjgit/GameZone/tree/main/Games/Drum_kit)|
 | [Prime-No-Finder-Game] (https://github.com/kunjgit/GameZone/tree/main/Games/Prime-No-Finder-Game)|  
 | [Synonym-Typing-Game] (https://github.com/kunjgit/GameZone/tree/main/Games/Synonym-Typing-Game)|
+| [Color Blink Challenge] (https://github.com/kunjgit/GameZone/tree/main/Games/Color Blink Challenge)|
```

Comment:
Follow  naming convention while naming game folder and respectively change the game folder after that here

Reply:
@ishita-43  can you please confirm ? I have done the changes.